install/install.sh

```diff
@@ -432,4 +432,9 @@ info "Please follow the steps for your shell to complete the installation:
   Add the following to the end of ${BOLD}~/.config/ion/initrc${NO_COLOR}:
 
       eval \$(starship init ion)
+
+  ${BOLD}${UNDERLINE}Tcsh${NO_COLOR}
+  Add the following to the end of ${BOLD}~/.tcshrc${NO_COLOR}:
+
+      eval "`starship init tcsh`"
```

Comment:
Here is what `shellcheck` thinks about it:
```
Line 439:
      eval "`starship init tcsh`"
            ^-- SC2046: Quote this to prevent word splitting.
            ^-- SC2006: Use $(...) notation instead of legacy backticked `...`.
```

First of all AFAIK, you don't need to quote _backticked_ portion so it can be simplified to:
```tcsh
eval `starship init tcsh`
```

and to incorporate that nicely into the existing script use the following form:
```
  ${BOLD}${UNDERLINE}Tcsh${NO_COLOR}
  Add the following to the end of ${BOLD}~/.tcshrc${NO_COLOR}:

      eval \`starship init tcsh\`
```

Escaping backticks prevents SC2006 false positive.

Reply:
Thanks, I have submitted a PR: #2429, to fix these issues